app/services/bill-runs/two-part-tariff/fetch-review-licence-results.service.js

```diff
@@ -39,6 +41,12 @@ async function _fetchBillRun (billRunId) {
     })
 }
 
+async function _licenceRef (licenceId) {
+  return LicenceModel.query()
```

Comment:
would it be better to extract the licenceRef here in the function and return it rather than having `licenceRef.licenceRef` ?